python-tests/__snapshots__/test_templates.ambr

```diff
@@ -105,21 +105,13 @@
                   
                     
                       <li class="govuk-footer__list-item">
-                        <a class="govuk-footer__link" href="https://www.gov.uk/guidance/digital-outcomes-and-specialists-suppliers-guide">
+                        <a class="govuk-footer__link" href="https://www.gov.uk/guidance/how-to-apply-to-sell-on-the-digital-outcomes-framework">
                           Applying to sell on the DOS framework
                         </a>
                       </li>
                     
                   
                     
-                      <li class="govuk-footer__list-item">
-                        <a class="govuk-footer__link" href="https://www.gov.uk/guidance/how-to-sell-your-digital-outcomes-and-specialists-services">
-                          Responding to buyer requirements on the DOS framework
-                        </a>
-                      </li>
```

Comment:
Are we sure that we want to remove this from DMP 1.0? DOS5 is still live, so suppliers are still responding to buyer requirements.

Reply:
Thanks for pointing this out, I hadn't realised I'd made an assumption from the ticket (but I had). Ticket doesn't specifically mention but it does say that the content is no longer applicable, so it makes sense that this is an 1.5 exclusive change - (my assumption being it was required in both 1.0 and 1.5).